### PR TITLE
Gatsby v2

### PR DESCRIPTION
**Pull request by Bouncey**
This PR updates `gatsby` to version 2.

Thanks to @ValeraS for all of his hard work.

@QuincyLarson for visibility

### CODE REVIEW

**Comment by Bouncey** [comment]
Wow! `frame-runner.js` has been reduced by over 65%! Brilliant!

**Comment by Bouncey** [comment]
@ValeraS can you rebase this PR against `master` please

File: package.json [hunk 1]
```diff
@@ -4,61 +4,55 @@
...
-    "date-fns": "^1.29.0",
     "debug": "^3.1.0",
     "dotenv": "^5.0.1",
     "enzyme": "^3.3.0",
-    "enzyme-adapter-react-15": "^1.0.5",
+    "enzyme-adapter-react-16": "^1.5.0",
     "fetchr": "^0.5.37",
-    "gatsby": "^1.9.243",
-    "gatsby-link": "^1.6.39",
+    "gatsby": "next",
```
Comment (Bouncey) [inline package.json]: Could we pin `gatsby` and related plugins to the latest release, just to guard against API changes that happen in the beta phase that start breaking things for us.

Comment (ValeraS) [reply package.json]: yarn.lock does it, doesn't it?

Comment (Bouncey) [reply package.json]: That is correct, I forgot about yarn in this repository 👍

**Comment by raisedadead** [comment]
This has landed in the main repository via https://github.com/freeCodeCamp/freeCodeCamp/pull/18188

Nice work @ValeraS !

**Pull request closed without merging**